### ISSUE TITLE
Update Velohero to sync.

### ISSUE THREAD

**Issue by Joern-R** [GoldenCheetah/GoldenCheetah#2475]
Velohero site is offering an API to list and download single activities. So Sync. should be possible here as well.

**Comment by liversedge** [comment]
I changed to P1 - they are the changes we highlight in the release notes. 
P2 - less important, but not cosmetic
P3 - cosmetic / niceties 

P0 are reserved for crash fixes.

I should document this somewhere !!

**Comment by liversedge** [comment]
Just trying to set the burn down list for 3.5 for end of the year -- would you like me to move this to release 3.6 for next year?

**Comment by Joern-R** [comment]
Yes 3.6 is fine - maybe we will even skip it. There seems to be very limited use of the service in combination with GoldenCheethah. I changed the milestone.

**Comment by grauser** [comment]
I can take this issue if @Joern-R is OK.

**Comment by amtriathlon** [comment]
I am closing this since the notice of end of service on May 1, 2026 on https://www.velohero.com/

We should remove the current integration for the next version too.
